mappings/net/minecraft/client/render/entity/WitherEntityRenderer.mapping

```diff
@@ -1,3 +1,3 @@
 CLASS net/minecraft/class_964 net/minecraft/client/render/entity/WitherEntityRenderer
-	FIELD field_4812 INVINCIBLE_SKIN Lnet/minecraft/class_2960;
-	FIELD field_4813 SKIN Lnet/minecraft/class_2960;
+	FIELD field_4812 INVINCIBLE_TEXTURE Lnet/minecraft/class_2960;
```

Comment:
Or do we want to do INVULNERABLE to match some of the method names in WitherEntity

Reply:
Invulnerable is better since that's what the texture is called.

Reply:
```suggestion
	FIELD field_4812 INVULNERABLE_TEXTURE Lnet/minecraft/class_2960;
```